apps/omg_performance/lib/omg_performance/byzantine_events.ex

```diff
@@ -14,130 +14,248 @@
 
 defmodule OMG.Performance.ByzantineEvents do
   @moduledoc """
-  OMG network child chain server byzantine event test entrypoint. Setup and runs performance byzantine tests.
+  OMG network child chain server byzantine event test entrypoint. Runs performance byzantine tests.
 
-  # Usage
+  ## Usage
 
-  See functions in this module for options available
+  To setup, once you have your Ethereum node and a child chain running, from a configured `iex -S mix run --no-start`
+  shell do:
```

Comment:
Do you mean repl here? I believe it's common term
```suggestion
  REPL do:
```

Please replace other occurences :point_down:

Reply:
but, but https://hexdocs.pm/iex/IEx.html says shell and it's more common according to google :P